.golangci.yml

```diff
@@ -8,6 +8,7 @@ linters:
     - bidichk
     - bodyclose
     - contextcheck
+    - copyloopvar
```

Comment:
It's not clear to me that you should add `copyloopvar` and change the code until the `go.mod` declares go 1.22.x?

https://go.dev/blog/loopvar-preview

> To ensure backwards compatibility with existing code, the new semantics will only apply in packages contained in modules that declare go 1.22 or later in their go.mod files.

Reply:
Yeah, just doing some proactive maintenance here. The code changes here are safe for Go 1.21 since there's no `t.Parallel` in play... and with Go 1.22 that whole pattern will thankfully be a thing of the past. The linters are both smart enough to remain dormant until the module adopts Go 1.22.